client/src/components/podcasts/PodcastForm.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react"
+import { Redirect } from "react-router-dom"
+import ErrorList from "../ErrorList.js"
+import translateServerErrors from "../../services/translateServerErrors.js"
+
+const PodcastForm = (props) => {
+  const [podcastRecord, setPodcastRecord] = useState({
+    name: "",
+    description: "",
+  })
+  const [errors, setErrors] = useState([])
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+
+  const handleInputChange = (event) => {
+    setPodcastRecord({
+      ...podcastRecord,
+      [event.currentTarget.name]: event.currentTarget.value,
+    })
+  }
+
+  // const fieldReset = () => {
+  //   setPodcastRecord({
+  //     name: "",
+  //     description: "",
+  //   })
+  // }
```

Comment:
haha comment this in and use it! haha might be a relic from having it ready to present on it ʕ•ᴥ•ʔ